std/zig/render.zig

```diff
@@ -717,24 +717,55 @@ fn renderExpression(
                     };
 
                     if (maybe_row_size) |row_size| {
+                        // A place to store the width of each expression and its column's maximum
+                        var widths = try allocator.alloc(usize, exprs.len + row_size);
+                        defer allocator.free(widths);
+                        mem.set(usize, widths, 0);
+
+                        var expr_widths = widths[0 .. widths.len - row_size];
+                        var column_widths = widths[widths.len - row_size ..];
+
+                        // A scratch buffer used to find an expression's printed width
+                        var expr_width_buf = try std.Buffer.initSize(allocator, 0);
+                        defer expr_width_buf.deinit();
+                        var expr_width_stream = &std.io.BufferOutStream.init(&expr_width_buf).stream;
```

Comment:
Doesn't seem like we're using `expr_width_buf`  for anything other than counting the length of an item, so maybe we should use `CountingOutStream` and `NullOutStream` instead. See #1722

Reply:
Brilliant, 5 thumbs up :hatching_chick:

Reply:
Holy cannoli, removing that allocation made the tests pass too.